src/Service/ActivityPubManager.php

```diff
@@ -157,38 +157,40 @@ public function findActorOrCreate(?string $actorUrlOrHandle): null|User|Magazine
             return $this->userRepository->findOneBy(['username' => $name]);
         }
 
+        $user = $this->userRepository->findOneBy(['apPublicUrl' => $actorUrl]);
```

Comment:
It seems `findOneBy` is breaking with this change to `'apPublicUrl'` instead of `'apProfileId'`, I switched it back to `'apProfileId'` and now my microblog section is working.

Reply:
it's also why i was getting a bunch of these errors... the mastodon users already exist, it just couldn't find them since several users (~20k) have their 'apPublicUrl' set to null, so it's trying to create them again..

```
21:00:24 WARNING   [messenger] Error thrown while handling message App\Message\ActivityPub\Inbox\ActivityMessage. Sending for retry #3 using 4800000 ms delay. Error: "Handling "App\Message\ActivityPub\Inbox\ActivityMessage" failed: An exception occurred while executing a query: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "user_ap_id_idx"\nDETAIL:  Key (ap_id)=(AlexDafonte@red.niboe.info) already exists." ["class" => "App\Message\ActivityPub\Inbox\ActivityMessage","retryCount" => 3,"delay" => 4800000,"error" => """  Handling "App\Message\ActivityPub\Inbox\ActivityMessage" failed: An exception occurred while executing a query: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "user_ap_id_idx"\n  DETAIL:  Key (ap_id)=(AlexDafonte@red.niboe.info) already exists.  ""","exception" => Symfony\Component\Messenger\Exception\HandlerFailedException { …}]